<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YesOrNo</class>
 <widget class="QWidget" name="YesOrNo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="YN_Y">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>122</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="YN_MSG">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>52</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提示信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="YN_N">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>122</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
